package/gluon-status-page/javascript/status-page.js

```diff
@@ -286,42 +286,44 @@
 			'highlight': false,
 
 			'resize': function(w, h) {
-				var lastImage;
+				let lastImage;
 				try {
 					lastImage = ctx.getImageData(0, 0, w, h);
-				} catch (e) {}
+				} catch (e) {
+					console.error(e);
```

Comment:
I'm not sure whether this should be logged. Unfortunately, I don't remember why I added the try/catch here...

MDN describes two possible exceptions: https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/getImageData#exceptions

I don't think we can hit the SecurityError, so maybe this was just an ugly way to catch `w` or `h` being zero?